ojd_daps_skills/pipeline/extract_skills/extract_skills.py

```diff
@@ -301,7 +301,10 @@ def map_skills(self, predicted_skills):
                             ],
                         )
                     )
-                    experience_list = predicted_skills[ix]["EXPERIENCE"]
+                    try:
+                        experience_list = predicted_skills[ix]["EXPERIENCE"]
+                    except:
+                        experience_list = []
 
```

Comment:
I hope this is ok now since predicted skills should always be in the format `{label: [] for label in self.labels}` i.e. `predicted_skills[ix]["EXPERIENCE"]` will equal `[]` anyway if there is no `EXPERIENCE`s